Source/Csla.Blazor/State/StateManager.cs

```diff
@@ -39,12 +39,14 @@ public Task InitializeAsync(TimeSpan timeout)
     /// Get state from cache.
     /// </summary>
     /// <param name="timeout">Time to wait before timing out</param>
-    private async Task GetState(TimeSpan timeout)
+    private async Task<Session> GetState(TimeSpan timeout)
```

Comment:
This method is only used by the initialization methods which do not return the session.
So instead of adding it here to the method signature the right thing to do would be to just discard the `RetrieveSession` result like: 
`_ = await _sessionManager.RetrieveSession(timeout);`
That would simplify the method a bit.